docs/guides/integration/docker.md

```diff
@@ -110,6 +110,18 @@ In either case, it is best practice to pin to a specific uv version, e.g., with:
 COPY --from=ghcr.io/astral-sh/uv:0.5.24 /uv /uvx /bin/
 ```
 
+!!! tip
+
+    While the Dockerfile example above pins to a specific tag, it's also
+    possible to pin a specific SHA256. Pinning a specific SHA256 is considered
+    best practice in environments that require reproducible builds as tags can
+    be moved across different commit SHAs.
+
+    ```Dockerfile
+    # The sha256 for 0.5.24
```

Comment:
```suggestion
    # e.g., using a hash from a previous release
```

Reply:
Unfortunately we'll auto-update 0.5.24 -> 0.5.25 on the next release but don't have a way to bump the SHA